Core/CoreTests/Dashboard/K5/Model/K5ImportantDateItemTests.swift

```diff
@@ -0,0 +1,80 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2022-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import XCTest
+import SwiftUI
+@testable import Core
+
+class K5ImportantDateItemTests: CoreTestCase {
+
+    let context = AppEnvironment.shared.globalDatabase.viewContext
+    var calendarEvents: [CalendarEvent]!
+    var importantDate: K5ImportantDate!
+
+    override func setUp() {
+        super.setUp()
+        let assignment = APICalendarEvent.make(
+            id: "1",
+            html_url: URL(string: "https://canvas.instructure.com/assignments/1")!,
+            title: "Important assignment title",
+            start_at: Date(fromISOString: "2022-01-03T08:00:00Z"),
+            type: .assignment,
+            context_name: "Test subject name"
+        )
+        let event = APICalendarEvent.make(
+            id: "2",
+            html_url: URL(string: "https://canvas.instructure.com/events/2")!,
+            title: "Important event title",
+            start_at: Date(fromISOString: "2022-01-03T09:00:00Z"),
+            type: .event,
+            context_name: "Test event name"
+        )
+        calendarEvents = [CalendarEvent.make(from: assignment, in: context),
```

Comment:
The type of `calendarEvents` is known so we can simply use `.make(` and omit `CalendarEvent`.